### PR TITLE
Print environment variables in every execution

### PR DESCRIPTION
**Pull request by meteorcloudy**
This will help us debug the failing projects on CI

### CODE REVIEW

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -702,6 +704,12 @@ def print_bazel_version_info(bazel_binary):
     execute_command([bazel_binary, "--nomaster_bazelrc", "--bazelrc=/dev/null", "info"])
 
 
+def print_environment_variables_info():
+    print_collapsed_group(":information_source: Environment Variables")
+    for key, value in os.environ.items():
+        eprint(key + "=" + value)
```
Comment (laszlocsomor) [inline buildkite/bazelci.py]: What do you think about printing the value in parentheses, so we can see if its empty? Like so:
```
        eprint("%s=(%s)" % (key, value))
```

File: buildkite/bazelci.py [hunk 2]
```diff
@@ -702,6 +704,12 @@ def print_bazel_version_info(bazel_binary):
     execute_command([bazel_binary, "--nomaster_bazelrc", "--bazelrc=/dev/null", "info"])
 
 
+def print_environment_variables_info():
+    print_collapsed_group(":information_source: Environment Variables")
+    for key, value in os.environ.items():
```
Comment (laszlocsomor) [inline buildkite/bazelci.py]: Also, please sort this dict.

Comment (meteorcloudy) [reply buildkite/bazelci.py]: Just tested, this dict is already sorted.

File: buildkite/bazelci.py (hunk 1 above)
Comment (meteorcloudy) [reply buildkite/bazelci.py]: Sounds good, done!

**Pull request merged**